integration-tests/metrics.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -e -u -o pipefail
+
+test_name="$(basename "${0}" .sh)"
+project='metrics'
+repository='https://github.com/dropwizard/metrics.git'
+revision='v5.0.0-rc22'
+additional_build_flags=''
+additional_source_directories=''
+shared_error_prone_flags='-XepExcludedPaths:.*/target/generated-sources/.*'
+# XXX: These don't work...
+patch_error_prone_flags='-Xep:CollectorMutability:OFF -XepOpt:Refaster:NamePattern=^(?!FileRules\$).*'
```

Comment:
This is a problem.. I'm not sure if you have an idea about this one? It will not actually filter out the `FileRules` Refaster rules...